AutoRest/Generators/CSharp/Azure.CSharp/TemplateModels/AzureParameterTemplateModel.cs

```diff
@@ -31,5 +32,20 @@ public override string DeclarationExpression
                 return base.DeclarationExpression;
             }
         }
+
+        public override string ParameterAccessor
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(this.ParameterGroup))
+                {
+                    return base.ParameterAccessor;
+                }
+                else
+                {
+                    return CodeNamer.CamelCase(this.ParameterGroup) + CodeNamer.PascalCase(base.ParameterAccessor);
```

Comment:
I might misunderstand it, but I would assume we should have a "." between them to make it a property getter expression.